kentikapi/client.go

```diff
@@ -200,3 +201,21 @@ func makeRetryInterceptor(c Config) grpc.UnaryClientInterceptor {
 		grpcretry.WithMax(*c.RetryCfg.MaxAttempts+1),
 	)
 }
+
+func makeLoggerInterceptor(c Config) grpc.UnaryClientInterceptor {
+	return func(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn,
+		invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
+		c.LogPayloads = true
```

Comment:
Excuse me what the puck?

Reply:
LogPayloads should not be modified here I guess?

Reply:
Done